test/Test/Database/LSMTree/Internal/RawPage.hs

```diff
@@ -230,118 +229,68 @@ prop_ops (PageContentFits kops) =
     fromOp Delete {}  = 2
     fromOp Mupsert {} = 1
 
+prop_rawPageOverflowPages :: PageContentFits -> Property
+prop_rawPageOverflowPages (PageContentFits kops) =
+    rawPageOverflowPages page === length overflowPages
+  where
+    (page, overflowPages) = toRawPage (PageContentFits kops)
+
+prop_fromToReferenceImpl :: PageContentFits -> Property
+prop_fromToReferenceImpl (PageContentFits kops) =
+    -- serialise using the reference impl
+    -- deserialise using the real impl and convert back to the reference types
+        fromRawPage (toRawPage (PageContentFits kops))
```

Comment:
I would like it to be able to distinguish functions from real and reference impl more easily, not just through comments. Maybe do a qualified import `as Ref`?

In this case I have the comment and even without it would probably think that `toRawPage` comes from the reference implementation, since `PageContentFits` is a type the real implementation should not know about. But I don't really see where it comes from (no import list either), so it might come from some extra module and do something else. So IMO would still be an improvement to clearly label it.

Reply:
Done. I also wrapped lines and added an `IsString` instance for `RawBytes` so we can write
```
(SerialisedKey "\x52\x53")
```
rather than
```
(SerialisedKey $ RB.pack [0x52, 0x53])
```